org.mixedrealitytoolkit.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyboardTouchAdapter.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clauseusing System;
+
+using Microsoft.MixedReality.GraphicsTools;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace MixedReality.Toolkit.UX.Experimental
+{
+    public class NonNativeKeyboardTouchAdapter : MonoBehaviour
+    {
+        private void Awake()
+        {
+            var defaultAudioComponent = GetComponent<AudioSource>();
+            defaultAudioComponent.playOnAwake = false;
+            defaultAudioComponent.spatialize = false;
```

Comment:
I don't think we should be explicitly disabled spatial audio here.   Why should allow the prefab to define this.

Reply:
Done